### ISSUE TITLE
Lower slider starts off-screen?

### ISSUE THREAD

**Issue by simongis** [ArcGIS/c-through#6]
Its late, so forgive me if this is a basic one, but when I open either your demo site or a fork on my own web server, the lower right controls appear outside of the initial viewport and not aligned to the bottom of the scene.    Is this an easy fix?  Would like to show this at an upcoming 3D conference.

![11](https://user-images.githubusercontent.com/2769383/31122727-acedf06e-a888-11e7-8bb9-ea1b632f7825.jpg)

**Comment by ialixandroae** [comment]
Hi @simongis , i think it`s a browser/resolution problem. If you zoom out in your browser to 75% it should be fine. I, myself, have this problem with laptops with high DPI which have that windows option to increase tabs/icons etc to 125% or so...

Ionut

**Comment by gutierrezj** [comment]
Right, the c-through application is not designed/implemented to be responsive to different browser resolutions and aspect ratios (it was really just a prototype). However this should not be hard to fix if you pull the source code.
